TAF_EasyRest/TestFramework/PageComponents/UpapprovedRestaurantPageComponent .cs

```diff
@@ -0,0 +1,29 @@
+﻿
+namespace TestFramework.PageComponents
+{
+    public class UnapprovedRestaurantPageComponent
+    {
+        private IWebDriver driver;
+
+        public UnapprovedRestaurantPageComponent(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        private IWebElement _approveButton => driver.FindElement(By.XPath("//span[text() = 'Approve']/parent::button)]"));
+        private IWebElement _disapproveButton => driver.FindElement(By.XPath("//span[text() = 'Approve']/parent::button)]"));
+
+
+        public UnapprovedRestaurantPageComponent ClickApproveButton()
+        {
+            _approveButton.Click();
+            return this;
```

Comment:
Do you really have the same page when clicking the Approve or Cancel button?
I believe clicking on these buttons should somehow change the Dom tree of the page; otherwise, according to the implemented logic, it doesn't matter what I'm clicking on; it always returns the same component over which I can click on the Approve button over and over again unlimited time. It completely breaks the meaning of the term "class" and OOP principles because you provide the abilities to the object that isn't reflected on the actual page.